en/installation/README.md

```diff
@@ -14,38 +14,68 @@ If you are attending one of the [Django Girls events](https://djangogirls.org/ev
 - If your workshop does not have an installation party (or you couldn't attend), and if the organizers didn't ask you to try to install everything before you arrived, skip this page and go straight to the [How the Internet works](../how_the_internet_works/README.md) chapter. You'll be installing everything you need as you work through the tutorial.
 
 # Installation
-In this tutorial you will be building a blog. In order to do that, as you go through the tutorial you'll be instructed on how to install various software on your computer and set up some online accounts as they are needed. This page gathers all of the installation and sign-up instructions in one place (which is useful for some workshop formats).
+In this tutorial you will be building a blog. In order to do that, as you go through the tutorial you'll be instructed on how to
+install various software on your computer and set up some online accounts as needed (if you are using local desktop environment) or instructed on how to create online accounts only (if you are using cloud development). This page gathers all of the installation and sign-up instructions in one place (which is useful for some workshop formats).
 
-<!--sec data-title="Chromebook setup (if you're using one)"
-data-id="chromebook_setup" data-collapse=true ces-->
-{% include "/chromebook_setup/instructions.md" %}
-<!--endsec-->
+## Local Installation
```

Comment:
Move this underneath Cloud Development. This way, RunCode is first (because it's the easiest) then local, which is split into the operating system instructions